src/lightning_app/utilities/openapi.py

```diff
@@ -0,0 +1,56 @@
+import json
+from typing import Any, Dict
+
+
+def _duplicate_checker(js):
```

Comment:
Mind adding docstring to those utlities.

Reply:
done